project1-sps/script.js

```diff
@@ -0,0 +1,155 @@
+//Objective SPS human vs computer
+// What is the input going to be? scissors, paper, stone
+// What should the output be?
+//scissors beats paper,
+//paper beats stone, and
+//stone beats scissors
+//same will be draw
+// How will the computer's option be randomly generated? math.random
+// How many different cases are there?
+// Scissors Paper Stone
+// 0        0     0 (not possible)
+// 0        0     1 (YES, BOTH STONE)
+// 0        1     0 (YES, BOTH PAPER)
+// 0        1     1 (YES)
+// 1        0     0 (YES, BOTH SCISSORS)
+// 1        0     1 (YES)
+// 1        1     0 (YES)
+// 1        1     1 (not possible)
```

Comment:
very nice pseudo code to outline your strategy and thought process before diving into code. keep it up for your future projects!